mcp/cmd.go

```diff
@@ -13,10 +13,19 @@ import (
 	"time"
 )
 
+const (
+	defaultTerminateDuration = 5 * time.Second
+	minTerminateDuration     = 1 * time.Second
+)
+
 // A CommandTransport is a [Transport] that runs a command and communicates
 // with it over stdin/stdout, using newline-delimited JSON.
 type CommandTransport struct {
 	Command *exec.Cmd
+	// TerminateDuration controls how long Close waits after closing stdin
+	// for the process to exit before sending SIGTERM.
+	// If less than 1 second (including zero or negative), the default of 5s is used.
```

Comment:
I don't think there should be a minimum. 0 should be 5s, but 1 (nanosecond) should be allowed, in case someone doesn't want to wait.